### PR TITLE
Increase error level from warning to error

### PR DESCRIPTION
**Pull request by DanVanAtta**

### CODE REVIEW

File: src/main/java/games/strategy/net/nio/NioReader.java [hunk 1]
```diff
@@ -146,7 +146,7 @@ private void loop() {
       } catch (final Exception e) {
         // catch unhandles exceptions to that the reader
         // thread doesnt die
-        logger.log(Level.WARNING, "error in reader", e);
+        logger.log(Level.SEVERE, "error in reader", e);
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/net/nio/NioReader.java]: What about using logger.warn?

Comment (DanVanAtta) [reply src/main/java/games/strategy/net/nio/NioReader.java]: We're still using the java.util api here and not a logback api regrettable. So there is no 'warn' method, there is a 'warrning' method, but it also does not take an exception as an arg so we woudl lose the stack trace.

Comment (RoiEXLab) [reply src/main/java/games/strategy/net/nio/NioReader.java]: Meant warning, but you are right. The stacktrace is valueable

Comment (RoiEXLab) [reply src/main/java/games/strategy/net/nio/NioReader.java]: Also it should be the severe method, but same problem...

**Pull request merged**